ArduCopter/AP_Arming.cpp

```diff
@@ -32,20 +32,20 @@ bool AP_Arming_Copter::pre_arm_checks(bool display_failure)
         return true;
     }
 
-    // check if motor interlock and Emergency Stop aux switches are used
+    // check if Throttle Hold and Emergency Stop aux switches are used
     // at the same time.  This cannot be allowed.
-    if (rc().find_channel_for_option(RC_Channel::AUX_FUNC::MOTOR_INTERLOCK) &&
+    if (rc().find_channel_for_option(RC_Channel::AUX_FUNC::THROTTLE_HOLD) &&
         rc().find_channel_for_option(RC_Channel::AUX_FUNC::MOTOR_ESTOP)){
-        check_failed(ARMING_CHECK_NONE, display_failure, "Interlock/E-Stop Conflict");
+        check_failed(ARMING_CHECK_NONE, display_failure, "Throttle Hold/E-Stop Conflict");
         return false;
     }
 
-    // check if motor interlock aux switch is in use
+    // check if Throttle Hold aux switch is in use
     // if it is, switch needs to be in disabled position to arm
     // otherwise exit immediately.  This check to be repeated,
     // as state can change at any time.
     if (copter.ap.using_interlock && copter.ap.motor_interlock_switch) {
-        check_failed(ARMING_CHECK_NONE, display_failure, "Motor Interlock Enabled");
+        check_failed(ARMING_CHECK_NONE, display_failure, "Throttle Hold Engaged");
```

Comment:
This should actually be "Throttle Hold Off". Throttle hold means that the throttle can't advance, or is inactive. If throttle hold is off, then the throttle is active.